### PR TITLE
Sep 11 + Sep 13 + Sep 15 Changelog

### PR DESCRIPTION
**Pull request by ashleygmulligan2**

### CODE REVIEW

File: changelog/platform.mdx [hunk 1]
```diff
@@ -4,6 +4,123 @@ description: "Additions and updates to the Flatfile platform"
...
+    <Snippet file="chips/files.mdx" />
+    <div
+      style={{
+        float: "left",
+        display: "table-column",
+        paddingLeft: "30px",
+        width: "calc(80% - 30px)",
+      }}
+    >
+        Increased mapping concurrency from 2 to 5 due to significant reductions in database load from mapping operations. This adjustment improves processing efficiency.
```
Comment (dboskovic) [inline changelog/platform.mdx:51]: I don't think this is meaningful to users

File: changelog/platform.mdx [hunk 2]
```diff
@@ -4,6 +4,123 @@ description: "Additions and updates to the Flatfile platform"
...
+    <Snippet file="chips/files.mdx" />
+    <div
+      style={{
+        float: "left",
+        display: "table-column",
+        paddingLeft: "30px",
+        width: "calc(80% - 30px)",
+      }}
+    >
+      When a file stream becomes unreadable due to file malformation,improvements were made to ensure that any errors are correctly communicated to the UI.
```
Comment (dboskovic) [inline changelog/platform.mdx:102]:
```suggestion
      When a file stream becomes unreadable due to file malformation, improvements were made to ensure that any errors are correctly communicated to the UI.
```

File: changelog/platform.mdx [hunk 3]
```diff
@@ -4,6 +4,174 @@ description: "Additions and updates to the Flatfile platform"
...
+      **Introducing: Code Blocks for Documents**
+
+      ![image](../changelog/images/documents_codeblocks.png)
+
+      We've introduced the ability to add code blocks to your Documents. You can now use the following syntax to define code blocks and specify the language:
+
+      ```js
+      ```js
+      // Your code goes here
+      ```
```
Comment (dboskovic) [inline changelog/platform.mdx:31]: Does this actually format right? The double ``` to start but not at the end?

Comment (ashleygmulligan2) [reply changelog/platform.mdx:31]: It wouldn't let me add the end or it tried to close it. Need to probably make this a picture actually

**Pull request merged**